doc/building_windows.md

```diff
@@ -0,0 +1,195 @@
+# Building Julius for Windows
+
+You have two main options in how you want to build Julius. You can either:
+
+* [Build using **Microsoft Visual Studio**](#a-using-microsoft-visual-studio-64-bit-build); or
+* [Install CMake and MinGW-w64 and **use the command line**](#b-using-cmake-and-mingw-w64-via-command-line)
+
+
+## A. Using Microsoft Visual Studio (64-bit build)
+
+1. Make sure you have [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/) installed. Please note that only Visual Studio 2017 (fully updated) or Visual Studio 2019 can be directly used.
+These instructions assume you are using Visual Studio 2019.
+
+2. Download the `SDL2` and `SDL2_mixer` development libraries, both as a zip file:
+
+	* [SDL2 dev libraries for Visual Studio](https://www.libsdl.org/release/SDL2-devel-2.0.10-VC.zip)
```

Comment:
I think it's better to link to the download page rather than the direct download.

Reply:
Ok, I'll do that.